### PR TITLE
Cardify team score information and add standings information to it

### PR DESCRIPTION
**Pull request by gjnelson112**

### CODE REVIEW

**Comment by gjnelson112** [comment]
Screen Shot 2023-02-05 at 5 43 55 PM
Screen Shot 2023-02-05 at 5 43 27 PM
Screen Shot 2023-02-05 at 5 43 38 PM

**Comment by gjnelson112** [comment]
the styling isn't perfect, but at least it's responsive?

Can revisit later, but a step in the right direction

**Comment by gjnelson112** [comment]
Screen Shot 2023-02-05 at 6 00 12 PM

update: 'NONE' now shows as '-'

File: models/user.py [hunk 1]
```diff
@@ -27,6 +27,33 @@ def get_id(self):
...
 
+    @classmethod
+    def standings(cls, dbsession):
+        users = dbsession.query(User).all()
+
+        users_w_total_score = {}
+        for user in users:
+            users_w_total_score[user] = user.team_score
+
+        users_sorted_by_score = sorted(users_w_total_score, key=lambda u: users_w_total_score[u])
```
Comment (ajpodell) [inline models/user.py:38]: if im reading this right, i dont think you need `users_w_total_score`. you can just do
`users_sorted_by_score = sorted(users, key=lambda u: u.team_score)` (or just reassign back to `users`)

**Review by ajpodell — APPROVED** [review]
left a pr into this branch you can merge if you like!

**Pull request merged**